balancer/balancer.go

```diff
@@ -371,3 +376,21 @@ type ClientConnState struct {
 // ErrBadResolverState may be returned by UpdateClientConnState to indicate a
 // problem with the provided name resolver data.
 var ErrBadResolverState = errors.New("bad resolver state")
+
+// A ProducerBuilder is a simple constructor for a Producer.  It is used by the
+// SubConn to create producers when needed.
+type ProducerBuilder interface {
+	// Build creates a Producer.  The first parameter is always a
+	// grpc.InvokerStreamer (a type to allow creating RPCs/streams on the
```

Comment:
I don't see the definition of this `grpc.InvokerStreamer` type in our existing codebase. Am I missing something? And what is this type used for?

Reply:
OOOPS.  This was a legacy name used for it when we were first creating it.  It's actually https://pkg.go.dev/google.golang.org/grpc#ClientConnInterface, and when I discovered this while writing the code, I forgot to go back and update this comment.

Reply:
Still not updated. Did you forget to push a commit?

Reply:
I didn't make the changes yet; I was waiting for your next pass before making these changes.